x-pack/plugins/timelines/public/components/actions/timeline/cases/add_to_case_action.tsx

```diff
@@ -263,6 +71,8 @@ const AddToCaseActionComponent: React.FC<AddToCaseActionProps> = ({
       updateCase: onCaseSuccess,
       userCanCrud: casePermissions?.crud ?? false,
       owner: [appId],
+      onClose: () =>
+        dispatch(tGridActions.setOpenAddToExistingCase({ id: eventId, isOpen: false })),
```

Comment:
Any issue with moving `closeCaseFlyoutOpen` up and using it here? And maybe we can keep the name a bit simpler with `closeCaseFlyout`?

Reply:
these are 2 different actions, `setOpenAddToExistingCase` here and `setOpenAddToNewCase`